src/design-system/scss/_variables.scss

```diff
@@ -414,7 +414,7 @@ $space-1-5: $space-baseline * 1.5 !default;
 $space-2: $space-baseline * 2 !default;
 $space-3: $space-baseline * 3 !default;
 $space-4: $space-baseline * 4 !default;
-$space-5: $space-baseline * 5 !default;
+$space-5: $space-baseline * 9 !default;
```

Comment:
this value probably shouldn’t be changed, or at least change the name of it

Reply:
removed this changes